lib/practice.md

```diff
@@ -3,58 +3,69 @@
 - What package did you install for enumeration?
 
 ```md
-<!-- your answer here -->
+npm install --save enumerationhere
 ```
 
 ## Research: Popular NPM Packages
 
 Package 1:
 ```md
-<!-- your answer here -->
+mocha
 ```
 1.  What problem does this package solve?
 ```md
-<!-- your answer here -->
+This package is similar to the rspec gem in ruby, which is used to test JS code.
 ```
 1.  How can I use this package in my own code? What steps are involved?
 ```md
-<!-- your answer here -->
+Install mocha package
+$ npm install mocha
+create test directory to store the test js file
+$ mkdir test
+
 ```
 1.  How well-maintained is this package? What criteria did I use to decide?
 ```md
-<!-- your answer here -->
+The most update commit in gitbuh is 20 days ago. The number of commits, and contributors are what I look for to decide.
 ```
 
 Package 2:
 ```md
-<!-- your answer here -->
+Moment
 ```
 1.  What problem does this package solve?
 ```md
-<!-- your answer here -->
+This package solves JS is not able to manipulate date.
```

Comment:
JavaScript can manipulate dates (see [Date - JavaScript | MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date)), but Moment.js does make it much easier.